src/eth_provider/starknet/kakarot_core.rs

```diff
@@ -214,7 +224,7 @@ mod tests {
             assert_eq!(tx.max_fee, FieldElement::ZERO);
 
             // Assert the length of calldata.
-            assert_eq!(tx.calldata.len(), transaction.transaction.length() + 6);
+            assert_eq!(tx.calldata.len(), (transaction.transaction.length() + 30) / 31 + 1 + 6);
```

Comment:
yes because now we pack the data in chunks of 31 (so we make a modulo 31 rounded up to the next integer as the chunks do) then we add 1 because the first element is simply `FieldElement::from(signed_data.len()`